test_data.rb

```diff
@@ -240,11 +240,13 @@ def test_localized_info_different_than_name
   end
 
   def test_suggestable_has_carrier
+    stations_to_ignore = %w(68192)
```

Comment:
Maybe this test is no longer needed if a station having a carrier is no longer a requirement?

Reply:
I think in general we want to enforce this - these stations are exceptions to the rule.

Reply:
Maybe instead of `stations_to_ignore` it could be `local_transport_stations` just to make it clear why they should be ignored? In the past I have found it hard to understand why some of these tests exist, often its down to business logic that hasn't been explained, so maybe a comment explaining this could be useful as well?

Reply:
:thinking: Is there any native renfe ID for this station?